mkdocs.yml

```diff
@@ -38,6 +43,10 @@ nav:
             - TMDB: developers/external_services/tmdb.md
             - Google Books: developers/external_services/gbooks.md
         - Database: developers/database.md
+        - Event Sourcing: developers/event_sourcing.md 
+        - SandBox: 
+            - Profile: developers/sandbox/profile.md
+            - Additional Content: developers/sandbox/additional_content.md
```

Comment:
Event sourcing part, should be move to: `Application Services > API` (?)
Sandbox part, should be renamed "Admin panel" and be move to `Application Services` (?)